multiaccounts/accounts/database.go

```diff
@@ -108,6 +108,7 @@ func (db Database) Close() error {
 	return db.db.Close()
 }
 
+// TODO remove photoPath from settings
```

Comment:
This is still a TODO or has been resolved?

Reply:
Still a TODO. I'm compiling a list of the TODOs in the description of this PR, then I'll make an issue for each. I thought it would be better to get the main body of this functionality into `develop` then incrementally improve it